app/android.c

```diff
@@ -241,11 +241,12 @@ void showFileOpen(JNIEnv* env, char* mimes) {
 	);
 }
 
-void showFileSave(JNIEnv* env) {
+void showFileSave(JNIEnv* env, char* mimes) {
     (*env)->CallStaticVoidMethod(
 		env,
 		current_class,
-		show_file_save_method
+		show_file_save_method,
+		mimesJString
```

Comment:
I think this is a compile error, isn't it missing:
`jstring mimesJString = (*env)->NewStringUTF(env, mimes);`